pkg/virtualization/core/storage/drive/dvddrive.go

```diff
@@ -18,3 +18,28 @@ func NewDvdDrive(instance *wmi.WmiInstance) (*DvdDrive, error) {
 	}
 	return &DvdDrive{wmivm}, nil
 }
+
+type DvdDriveCollection []*DvdDrive
+
+func NewDvdDriveCollection(instances []*wmi.WmiInstance) (col DvdDriveCollection, err error) {
```

Comment:
please follow the pattern for this repo.

https://github.com/microsoft/wmi/blob/242d2d5a9ed75e81d5c9eb4a56882e090d67e476/pkg/virtualization/core/storage/controller/scsicontrollercollection.go

Reply:
ie can you move to a separate go file